collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadList.java

```diff
@@ -126,6 +124,9 @@
     @Inject
     FormListDownloader formListDownloader;
 
+    @Inject
+    NetworkStateProvider networkStateProvider;
```

Comment:
While we're here could you rename the class to `FormDownloadListActivity`? Confuses me every time!

Reply:
Done.